### PR TITLE
RATEST-224:Adding report definition configuring

### PR DESCRIPTION
**Pull request by sherrif10**
This adds an app definition for reports configuration, then from this page its where we shall be able to use reporting app under system administration

https://issues.openmrs.org/browse/RATEST-224

### CODE REVIEW

**Comment by sherrif10** [comment]
cc @kdaud  waiting for review. Thanks. This configures the report app to be able to be accessible in openmrs server

File: ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java [hunk 1]
```diff
@@ -121,6 +121,10 @@ public FindPatientPage goToFindPatientRecord() {
         return new FindPatientPage(this);
     }
 
+    public ReportsPage goToReportsPage() {
+    	clickOn(REPORTS);
+    	return new ReportsPage(this);
+    }
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java:127]: Add an empty after line 127

File: ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class ManageAppsPage extends Page {
+	
+	private static final  By ADD_APP_DEFINITION = By.cssSelector("#content > button");
+	private static final  By APP_ID_FIELD = By.cssSelector("#appId-field");
+	private static final By  DEFINITION_JSON_FIELD = By.cssSelector("#json-field");
+	private static final String appIdField = "my.reportingui.reports";
+	private static final String definitionjsonField = "{"
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java:12]: Use the right case for the static final variables names

File: ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	}
+	
+	public void clickOnSaveButton() {
+		clickOn(SAVE_BUTTON);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/referenceapplication/manageApps.page";
+	}
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java:104]: Correspond ManageAppsPage indention with the indention of existing pages within the module

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java:104]: identation is the same as the whole project definition.Thanks

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java:103]: @sherrif10 have you had a look at the indention of the existing pages within the module?

**Comment by sherrif10** [comment]
We will not use this any where since we decided to carry out testing on legacyui Page. Am closing this.

**Pull request closed without merging**